flow/designs/nangate45/aes/config.mk

```diff
@@ -7,6 +7,7 @@ export SDC_FILE      = ./designs/$(PLATFORM)/$(DESIGN_NICKNAME)/constraint.sdc
 
 # These values must be multiples of placement site
 # x=0.19 y=1.4
-export DIE_AREA    = 0 0 620.15 620.6
-export CORE_AREA   = 10.07 11.2 610.27 610.8
+export DIE_AREA    = 0 0 249.85 250.6 
+export CORE_AREA   = 10.07 11.2 239.97 240.8 
```

Comment:
For non-macro designs I think it would be better to use utilization based initialization

Reply:
makes sense will change this -- I am keeping one design (typically gcd) with CORE/DIE settings as an example